Snippets/MetricsServiceControl/MetricsServiceControl_2/Configuration.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace MetricsServiceControl_2
+{
+    using NServiceBus;
+    using NServiceBus.Metrics.ServiceControl;
+
+    class Configuration
+    {
+        void ConfigureEndpoint(EndpointConfiguration endpointConfiguration)
+        {
+            #region SendMetricDataToServiceControl
+
+            endpointConfiguration.SendMetricDataToServiceControl(
+                serviceControlMetricsAddress: "address",
```

Comment:
Should we use the default queue value like in https://github.com/Particular/ServiceControl.Monitoring/pull/62